distributed/deploy/ssh.py

```diff
@@ -91,7 +91,12 @@ def __init__(
         self.nprocs = self.kwargs.pop("nprocs", 1)
 
     async def start(self):
-        import asyncssh  # import now to avoid adding to module startup time
+        try:
+            import asyncssh  # import now to avoid adding to module startup time
+        except ImportError:
+            raise ImportError(
+                "Dask SSHCluster requires `asyncssh`, please install it using pip or conda."
```

Comment:
Nitpick: just a small suggestion on the wording here (and in the other place we attempt to `import asyncssh`)

```suggestion
                "Dask's SSHCluster requires the `asyncssh` package to be installed. Please install it using pip or conda."
```